src/pages/signin/ThirdPartySignInPage.js

```diff
@@ -0,0 +1,136 @@
+
+    ...withLocalizePropTypes,
+
+    ...windowDimensionsPropTypes,
+};
+
+const defaultProps = {
+    account: {},
+    betas: [],
+    credentials: {},
+    signInProvider: 'google',
+};
+
+const capitalize = (word) => word.charAt(0).toUpperCase() + word.slice(1);
+
+function ThirdPartySignInPage(props) {
+    useEffect(() => {
+        Performance.measureTTI();
+
+        App.setLocale(Localize.getDevicePreferredLocale());
+    }, []);
+
+    const continueWithCurrentSession = () => {
+        console.log('ContinueWithCurrentSession');
+    };
+
+    const goBack = () => {
+        Navigation.navigate(ROUTES.HOME);
+    };
+
+    window.account = props.account;
+
+    return (
+        <SafeAreaView style={[styles.signInPage]}>
+            <SignInPageLayout
+                welcomeHeader={props.translate('welcomeText.getStarted')}
+                shouldShowWelcomeHeader
+            >
+                <Text style={[styles.mb5]}>{props.translate('thirdPartySignIn.alreadySignedIn', {email: 'johndoe@example.com'})}</Text>
+                <Button
+                    large
+                    text={props.translate('thirdPartySignIn.continueWithMyCurrentSession')}
+                    onPress={continueWithCurrentSession}
+                />
+                <Text style={[styles.mb5, styles.mt5]}>{props.translate('thirdPartySignIn.or')}</Text>
+                {props.signInProvider === 'google' && (
+                    <Button
+                        large
+                        text="Continue with Google"
+                        style={[styles.mb5]}
+                        onPress={continueWithCurrentSession}
+                    />
+                )}
+                <Text style={[styles.mt5]}>{props.translate('thirdPartySignIn.redirectToDesktopMessage')}</Text>
+                <Text style={[styles.mt5]}>{props.translate('thirdPartySignIn.goBackMessage', {provider: capitalize(props.signInProvider)})}</Text>
+                <TextLink
+                    style={[styles.link]}
+                    onPress={goBack}
+                >
+                    {props.translate('common.goBack')}.
+                </TextLink>
+                <Text style={[styles.textExtraSmallSupporting, styles.mt5, styles.mb5]}>
+                    {props.translate('thirdPartySignIn.signInAgreementMessage')}
+                    <TextLink
+                        style={[styles.textExtraSmallSupporting, styles.link]}
+                        href=""
+                    >
+                        {` ${props.translate('common.termsOfService')}`}
+                    </TextLink>
+                    {` ${props.translate('common.and')} `}
+                    <TextLink
+                        style={[styles.textExtraSmallSupporting, styles.link]}
+                        href=""
+                    >
+                        {props.translate('common.privacy')}
+                    </TextLink>
+                    .
```

Comment:
I think the dot should be part of a string if we need to have it, I don't think any strings should live outside the i18n list typically? Is there an example of this happening elsewhere in the app?